source/core/parameters_lagrangian.cc

```diff
@@ -598,6 +598,94 @@ namespace Parameters
       prm.leave_subsection();
     }
 
+    template <int dim>
+    void
+    ForceTorqueOnWall<dim>::declare_parameters(ParameterHandler &prm)
```

Comment:
We use _ to seperate words in function name instead of capital letters. The only place we use Capital Letters is for objects.

Reply:
Perfect